skietbaan_fe/src/components/LeaderboardPage.js

```diff
@@ -149,117 +220,168 @@ class LeaderboardPage extends Component {
+                                                    <div className="scrollableContainerC">
+                                                        {competitionsList}
+                                                    </div>
+                                                </div>
+                                                <div className="col-sm-4">
+                                                    <p style={pStyle}>Rankings in</p>
+
+                                                    
+                                                    <div className="scrollableContainerG">
+                                                       <div className="row justify-content-center" style={rankingIn}>
+                                                             Groups
+                                                       </div>
+                                                        {groupsList}
+                                                    </div>
+                                                </div>
                                             </div>
                                         </div>
-                                    </div>
+                                        <hr />
+                                    </Collapse>
                                 </div>
-                                <hr />
-                            </Collapse>
- 
+                            </div>
+                        </Collapse>
+                        <Collapse isOpened={this.closeMain(this.state.collapseFilter)}>
+                        <div className="row justify-content-center" style={ranktableStyle}>
+                            <table className="table">
+                                <thead >
+                                    <tr>
+                                        <td className="filterIconCOl" colSpan="1">
+                                            <div className="tableIcons">
+                                                <MDBBtn tag="a" size="lg" floating gradient="purple"
+                                                    onClick={this.onMouseClickFilter} >
+                                                    <img src={require('../resources/filter.png')} />
+                                                </MDBBtn>
+                                            </div>
+                                        </td>
+                                        <td className="LabelsCol" colSpan="3">
+                                            <div className="tableText">
+                                                <table className="tableTextContainer">
+                                                    <tr>
+                                                        <td className="GroupLabelCol" colSpan="3">
+                                                            {this.props.groups.length > 0 ? this.props.groups[this.state.selectedGroup].label : null}
```

Comment:
Spit over multiple lines

Reply:
m using the inline styles in classes with names that are identical to other but the i only want to modify that specific class with extra styles
can do that on the next pr, it will consume more time and the client want to see the demo this morning.